plugins/sms-twilio/src/manifest.json

```diff
@@ -64,7 +64,13 @@
     {
       "key": "event_invoice_add_draft",
       "label": "invoice.add_draft",
-      "description": "Example: Created draft of invoice for %%invoice.total%% %%invoice.currencyCode%%  for client %%client.companyName%%%%client.firstName%% %%client.lastName%%.",
+      "description": "Example: Created draft of invoice for %%invoice.total%% %%invoice.currencyCode%% for client %%client.companyName%%%%client.firstName%% %%client.lastName%%.",
+      "required": 0
+    },
+    {
+      "key": "event_invoice_draft_approved",
+      "label": "invoice.draft_approved",
+      "description": "Example: Approved draft of invoice for %%invoice.total%% %%invoice.currencyCode%% for client %%client.companyName%%%%client.firstName%% %%client.lastName%%.",
```

Comment:
We should have the invoice number available here right? If we do I would just use the same message as for `event_invoice_add`, since this is mostly meant for clients and they don't care about some drafts.